blocks/testimonial/index.js

```diff
@@ -0,0 +1,199 @@
+import './style.scss';
+import './editor.scss';
+
+const { __ } = wp.i18n;
+
+const {
+	registerBlockType,
+	BlockControls,
+	BlockAlignmentToolbar
+} = wp.blocks;
+
+const {
+	Toolbar
+} = wp.components;
+
+
+registerBlockType( 'rtgb/testimonial', {
+
+	title: __( 'Testimonial' ),
+	icon: 'format-status',
+	category: 'layout',
+	description: __( 'Display a testimonial with a picture, text, name and company' ),
+
+	attributes: {
+
+		image: {
+			type: 'object',
+			field: {
+				type: 'image',
+				buttonText: __( 'Upload' ),
+				removeButtonText: __( 'Remove' ),
+			},
+		},
+
+		content: {
+			type: 'array',
+			field: {
+				type: 'rich-text',
+				className: 'testimonial-content',
+				placeholder: __( 'Write testimonial content here' ),
+				tagName: 'div',
+			},
+			selector: '.testimonial-content',
+			source: 'children',
+		},
+
+		name: {
+			type: 'array',
+			field: {
+				type: 'rich-text',
+				className: 'testimonial-name',
+				placeholder: 'Name',
+				tagName: 'p',
+			},
+			selector: '.testimonial-name',
+			source: 'children',
+		},
+
+		companyName: {
+			type: 'array',
+			field: {
+				type: 'rich-text',
+				className: 'testimonial-company',
+				placeholder: 'Company',
+				tagName: 'p',
+			},
+			selector: '.testimonial-company',
+			source: 'children',
+		},
+
+		align: {
+			type: 'string',
+			default: 'none',
+		},
+
+		bgColor: {
+			type: 'string',
+			field: {
+				type: 'color',
+				label: 'Background Color',
+				// placement: 'inspector',
+			},
+		},
+
+		textColor: {
+			type: 'string',
+			field: {
+				type: 'color',
+				label: 'Text Color',
+				// placement: 'inspector',
+			},
+		}
+	},
+
+	getEditWrapperProps( attributes ) {
+		const { align } = attributes;
+		if ( 'full' === align ) {
```

Comment:
@yahilmadakiya We also need support for `wide` alignment.

Reply:
@manishsongirkar  I have added `wide`.